EIPS/eip-7708.md

```diff
@@ -20,24 +20,42 @@ Logs are often used to track when balance changes of assets on Ethereum. Logs wo
 
 ## Specification
 
-### Parameters
+### ETH transfer logs
 
-* `MAGIC`: `TBD`
+A log, identical to a LOG3, is issued for:
 
-### Functionality
+- Any transaction (including zero-value-transferring), before any other logs created by EVM execution
```

Comment:
Rationale being that something still happened on the wallet that may be of interest to the user. This is only on the outermost call.